### ISSUE TITLE
`list`?

### ISSUE THREAD

**Issue by zmoon** [johnnychen94/jill.py#88]
A `list` (sub)command to list the installed Julias managed by `jill` and where they are installed could be nice

**Comment by johnnychen94** [comment]
Agreed a `jill list` would be helpful. Currently, I just hit `julia<TAB>` and see what's there :)

I'd also want to introduce some ideas from `juliaup`: `jill switch <version>` that changes where the existing symlink targets to. For example `jill switch 1.5` that makes `julia` targets to `julia-1.5`
